### ISSUE TITLE
A question about a change in examples/pinn_forward/Lotka_Volterra.py

### ISSUE THREAD

**Issue by lijialin03** [lululxvi/deepxde#1437]
### Discussed in https://github.com/lululxvi/deepxde/discussions/1436

<div type='discussions-op-text'>

<sup>Originally posted by **lijialin03** August 15, 2023</sup>
Accroding to https://github.com/lululxvi/deepxde/pull/1158 parameters of training have been changed from red figure to green figure below.
<img width="530" alt="093798d2d5690fda3cf06b31652de50b" src="https://github.com/lululxvi/deepxde/assets/124568209/c20c5a0d-562d-44bb-8868-8ee51abad9c6">
<img width="552" alt="82a9cbe50e7712af2ffb3576c46cc289" src="https://github.com/lululxvi/deepxde/assets/124568209/cafb7294-8ead-4976-9d4f-6d0381a4953f">
This change will cause an error that loss probability appears ‘nan’ when training with LBFGS in paddle and pytorch backend (I only try these two). I think the reason is the example was not fully trained with Adam optimizer and at the same time, the initial learning rate of LBFGS was set too small (default learning rate of LBFGS optimizer of paddle and pytorch are 1).
So is this change necessary？</div>
